proxysql/Dockerfile

```diff
@@ -49,10 +49,17 @@ RUN set -ex; \
         procps-ng \
         perl-DBD-MySQL \
         perl-DBI \
-        tar; \
+        tar \
+        cpio; \
     microdnf clean all; \
     rm -rf /var/cache/dnf /var/cache/yum
 
+RUN set -ex; \
+    curl -Lf -o /tmp/server.rpm https://repo.percona.com/pxc-80/yum/release/8.0/RPMS/x86_64/percona-xtradb-cluster-server-8.0.28-19.1.el8.x86_64.rpm; \
```

Comment:
maybe we should use `$PXC_VERSION` env variable in the url?

Reply:
That line was just a temporary fix so that I can run the test and since the build team has fixed the issue by extracting the my_print_default from the server package and copying into docker I have now removed it.